Dockerfile.TLPSO-461

```diff
@@ -0,0 +1,73 @@
+FROM openjdk:8u131-jdk
+
+MAINTAINER Teaching and Learning <its.tl.dev@umich.edu>
+
+# TODO: set time zone to be MI?  Use NTP? See cpm (?)
+# TODO: secure properties / secrets
+# TODO: modify the entry point values for non-dev situations.
+
+#### Setup environment
+RUN apt-get update \
+ && apt-get install -y maven
+
+#### Get and build source
+WORKDIR /tmp
+
+## build the esbUtils (not directly available as a jar so build and install locally).
+#RUN git clone --branch v2.0 https://github.com/tl-its-umich-edu/esbUtils \
+RUN git clone --branch TLPSO-461 https://github.com/dlhaines/esbUtils \
+ && cd esbUtils \
+ && pwd \
+ && mvn clean install
+
+## build the SPE application
+COPY . /tmp
+# Don't run tests test on OpenShift build
+RUN mvn clean package -D maven.test.skip=true
+
+#### CLEAN UP container
+RUN apt-get remove -y maven git \
+    && apt-get autoremove -y
+
+RUN rm -rf ~/.m2
+#RUN df -h /
+
+############### assemble artifacts into /opt/spe #################
+
+RUN mkdir -p /opt/spe-bin
+RUN mv /tmp/target/spanish*jar /opt/spe-bin/spe.jar
+
+####### NOTE: security files will handled as OS secrets
+## install the configuration files
+
+RUN mkdir -p /opt/spe/config
+WORKDIR /tmp/config
+#RUN ls -l /tmp/config
+RUN cp /tmp/config/*properties /opt/spe/config/
+# don't insist that yml files exist.
+#RUN cp /tmp/config/*yml /opt/spe/config/; exit 0;
+RUN cp /tmp/config/*json /opt/spe/config/
+
+# Create directory to store persisted information.
+# It will be external storage.  SPE will automatically
+# manage the contents.
+RUN mkdir -p /opt/spe/persist
+
+# ### set default command to be the SPE jar.
+WORKDIR /opt/spe
+
+#RUN find /opt/spe -ls
+#RUN find /opt/spe-bin -ls
+
+# set entry point so can add arguments from "docker run" on command line.
+# EX: If start with:
+# docker run -e TZ=America/New_York spe_a --spring.profiles.include=ZOMBIE
+# ZOMBIE will be ADDED to the list of spring profiles to include.
+# 
+ENTRYPOINT ["java", "-jar","/opt/spe-bin/spe.jar", \
+            "-Djava.security.egd=file:/dev/./urandom", \
+            "--test.skipRun=false", \
+            "--spring.profiles.include=OS"\
```

Comment:
I see from the source code `config` directory lot of files start as `application -OS` and I see one of them is production file with `dev` and `qa`. how is the dev and qa file are not picked up during the production run? documentation is needed on this